app/uk/gov/hmrc/nisp/controllers/ExclusionController.scala

```diff
@@ -40,7 +41,13 @@ trait ExclusionController extends NispFrontendController with AuthorisedForNisp
   def showSP: Action[AnyContent] = AuthorisedByAny.async { implicit user => implicit request =>
     val nino = user.nino.getOrElse("")
     nispConnector.connectToGetSPResponse(nino).map {
-      case SPResponseModel(_, Some(spExclusions: ExclusionsModel)) => Ok(excluded_sp(nino, spExclusions))
+      case SPResponseModel(Some(spSummary: SPSummaryModel), Some(spExclusions: ExclusionsModel), niExclusionOption) => {
```

Comment:
@roundrobin99 The `case` is not dependent on the niExclusionOption having Some() or None, I need both. I've given it a name so I can check it in the code.